main.py

```diff
@@ -0,0 +1,80 @@
+import os
+import pandas as pd
+import dxpy as dx
+import datetime
+
+from util import read_yaml, dx_login, send_mail
+
+
+# define number of seqs
+seq = read_yaml('seq.yml')
+
+
+def main():
+
+    dx_login()
+
+    # defining directories in docker container
+    # Directories need to be mounted with docker run -v
+    GENETIC_DIR = 'var/genetics'
+    LOGS_DIR = 'var/log/dx-streaming-upload'
+
+    duplicates = []
+    final_duplicates = []
+    table_data = []
+
+    # loop through each file in /genetics/<SEQ> and /var/log/dx-stream-upload/<SEQ>
+    for file in seq:
+        gene_dir = '{}/{}'.format(GENETIC_DIR, file)
+        logs_dir = '{}/{}'.format(LOGS_DIR, file)
+
+        # get the duplicates between two directories /genetics & /var/log/.. with sets
+        temp_duplicates = set([x.strip() for x in os.listdir(gene_dir)]) & set([x.split('.')[1] for x in os.listdir(logs_dir)])
+
+        duplicates += list(temp_duplicates)
+
+    # find out if a run has been created for the project in dnanexus
+    for project in duplicates:
+        
+        # careful generator object only allows one read, similar to python3 map generator object
+        dxes = dx.search.find_projects(name="002_{}_\D+".format(project), name_mode="regexp", describe=True)
+        return_obj = list(dxes)
+
+        if return_obj:
+            proj_des = return_obj[0]['describe']
+            
+            table_data.append(
+                (project, datetime.datetime.fromtimestamp(proj_des['created'] / 1000.0).strftime('%Y-%m-%d'), '{} GB'.format(round(proj_des['dataUsage'])), proj_des['createdBy']['user'], proj_des['storageCost'])
```

Comment:
line length, try to stick to 79(ish) characters for code, this line doesn't fit in the github code review window so I can't see the end with scrolling horizontally. It's also a lot for one line, I'd probably preform the date since it's the longest then pass it in.

Maybe something like this:
```
created = datetime.datetime.fromtimestamp(
    proj_des['created'] / 1000.0).strftime('%Y-%m-%d'
)

table_data.append((
    project, created, f'{round(proj_des['dataUsage'])} GB',
    proj_des['createdBy']['user'], proj_des['storageCost']
))  
```

Reply:
yeah, that was a horrible line, will do some cosmetic + annotations to the whole script in the next push